integrations-and-supported-tools/sacred/notebooks/Neptune_Sacred.ipynb

```diff
@@ -37,7 +37,7 @@
         "id": "PTBMNOrWkg9l"
       },
       "source": [
-        "!pip install --q sacred==0.8.2 neptune-client[sacred]==0.9.18"
+        "!pip install --q sacred==0.8.2 neptune-client[sacred]==0.9.18 torch==1.9.0 torchvision==0.10.0"
```

Comment:
Sorry for doing this but with this new note on the #docs channel we've decided to drop the versions for neptune-client and neptune-INTEGRATION libraries so that we can test examples on updated client and push fixed on the product team when needed.